nautobot/extras/tests/swappie.py

```diff
@@ -0,0 +1,1435 @@
+from django.conf import settings
```

Comment:
What's this `swappie.py` file for?

Reply:
Good point, I didn't mean for that to get into the repo!